crates/solidity/outputs/npm/package/src/generated/index.d.ts

```diff
@@ -8,6 +8,15 @@
 
 /* auto-generated by NAPI-RS */
 
+export interface TextIndex {
```

Comment:
these two types need to be added to `crates/solidity/outputs/npm/package/src/syntax/nodes/index.js`

Reply:
1. That index.d.ts doesn't exist - not sure what we need there.
2. Cannot export these TS interfaces (pure types) in JS.

Reply:
1. Oops, I added an extra `/src/` there. The real file is in `crates/solidity/outputs/npm/package/syntax/nodes/index.d.ts`
2. In that case, we only need to export it from `index.d.ts`, not `index.js`?

Reply:
I've run slang_check to ensure the npm build, and the only *.d.ts I find is this generated one (with this comment), so I'm confused about this.

Yes, I think index.js doesn't need to be modified.